rest-server/src/templates/yarnContainerScript.mustache

```diff
@@ -64,7 +64,9 @@ docker run --name $docker_name \
   --volume /tmp/pai-root/alive/$APP_ID:/alive \
   --env FRAMEWORK_NAME=$FRAMEWORK_NAME \
   --env HADOOP_USER_NAME=$HADOOP_USER_NAME \
+  --env DEFAULT_HDFS_URI={{{ hdfsUri }}} \
   --env PAI_TASK_INDEX=$TASK_INDEX \
+  --env PAI_TASK_GPU_COUNT={{{ taskData.gpuNumber }}} \
```

Comment:
pls refer to `PAI_TASK_GPU_NUM` https://github.com/Microsoft/pai/blob/master/rest-server/src/templates/dockerContainerScript.mustache#L47

Reply:
Got it.